### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Elijahscriptdev**
Set up a simple form that will let users input their location and will fetch the weather info

### CODE REVIEW

File: src/getWeatherData.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    });
+
+    btn2.addEventListener('click', () => {
+      tempInCelius.style.display = 'block';
+      tempInFahrenheit.style.display = 'none';
+      btn1.style.display = 'block';
+      btn2.style.display = 'none';
+    });
+  } catch (err) {
+    // console.log(err);
```
Comment (bellom) [inline src/getWeatherData.js:44]: This should be a production code, please remove debugging statements. (`console.log`), even if that feature is not fully implemented, users should not see debugging messages.

File: src/getWeatherData.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+const weatherInfo = async (info) => {
+  try {
+    const weatherApi = `https://api.openweathermap.org/data/2.5/weather?q=${info}&APPID=2874e0623c8807994e18916c8cd78f21`;
```
Comment (bellom) [inline src/getWeatherData.js:3]: we need to make different requests for the Celsius data and the Fahrenheit data.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

File: src/search.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import weatherInfo from './getWeatherData';
+
+const searchBar = () => {
+  const searchBtn = document.getElementById('search-btn');
+  searchBtn.addEventListener('click', () => {
+    const info = document.getElementById('search-input').value;
+    weatherInfo(info);
```
Comment (bellom) [inline src/search.js:7]: Good code splitting 👍 
There should be some changes in the page appearance based on the received data(According to the Odin instructions, screenshot below; )
image

**Comment by Elijahscriptdev** [comment]
Please i want to create another pull request.
I am having issues with this one.
Thanks

**Comment by bellom** [comment]
> Please i want to create another pull request.
> I am having issues with this one.
> Thanks

Kindly tag a link to this PR for the next TSE's to see the previous review.

**Pull request merged**